### ISSUE TITLE
Filtering spectra within a particular RT range in each precursorMz? functions similar to `filterPrecursorMz`?

### ISSUE THREAD

**Issue by gmhhope** [jorainer/SpectraTutorials#8]
Could I easily implement filtering spectra within a particular RT range according to their character in the MS1 scan? Something similar to `filterPrecursorMz`?

Thanks,
Minghao Gong

**Comment by gmhhope** [comment]
I don't know whether this is the right one? Could you briefly comment on this? Thanks very much!
```
filterRt(sps_ms2, rt = rt_range)
```

Best,
Minghao

**Comment by jorainer** [comment]
Yes, exactly, `filterRt` is the function that lets you filter `Spectra` by a retention time range (independently if the spectrum is MS level 1 or 2).
